android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/WorkerBinder.kt

```diff
@@ -166,10 +193,62 @@ public object WorkerBinder {
   }
 }
 
+/**
+ * Worker.onStart/onStop duration information
+ */
+public data class WorkerBinderDuration(
```

Comment:
The naming "WorkerBinderDuration" feels a bit too implementation specific on duration. This is more of a worker metadata + callback. What do you think about naming it more generally?

Reply:
Yes, it makes sense. Will update in a bit

Reply:
If we like this pattern after rollout, maybe it makes sense to have an Interactor Metadata class too.

Reply:
@tyvsmith Updated the naming